nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/common/v1/ConanRoutes.groovy

```diff
@@ -157,12 +195,36 @@ class ConanRoutes
     )
   }
 
+  static Builder conanManifestProxy() {
+    new Builder().matcher(
+        and(
+            new ActionMatcher(GET, HEAD),
+            or(
+                new TokenMatcher("{other:.*}/{path:.*}/${STANDARD_FORM}/packages/{sha:.+}/conanmanifest.txt"),
+                new TokenMatcher("{other:.*}/{path:.*}/${STANDARD_FORM}/conanmanifest.txt")
+            )
+        )
+    )
+  }
+
+  static Builder conanManifestHosted() {
+    new Builder().matcher(
+        and(
+            new ActionMatcher(GET, HEAD),
+            or(
+                new TokenMatcher("{other:.*}/${STANDARD_FORM}/packages/{sha:.+}/conanmanifest.txt"),
+                new TokenMatcher("{other:.*}/${STANDARD_FORM}/conanmanifest.txt")
+            )
+        )
+    )
+  }
+
   private static TokenMatcher conanManifestMatcher() {
-    new TokenMatcher("/{path:.*}/${STANDARD_FORM}/conanmanifest.txt")
+    new TokenMatcher("{other:.*}/${STANDARD_FORM}/conanmanifest.txt")
```

Comment:
Other looks like unclear. It is better to name it something like: api_prefix

Reply:
not a bad idea.
Fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/91/commits/d633930a9a11be20a74dc8463401b809b5bfe5bd